lms/urls.py

```diff
@@ -685,6 +685,16 @@
     ),
 ]
 
+
+urlpatterns += [
+    url(
+        r'^bundles/{}/about'.format(
```

Comment:
**Question:** Why is `bundles` a top-level resource instead of including it within the `journals` resource?

Reply:
fixed, moved to journals app and beneath journals namespace